notebooks/point-cloud-processing.ipynb

```diff
@@ -382,28 +517,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 23,
    "metadata": {},
    "outputs": [],
    "source": [
-    "treetop_spheres.color = matplotlib.cm.Vega10(np.arange(len(lep['Z']))%10)[:,0:3]"
+    "treetop_spheres.color = matplotlib.cm.prism(np.arange(len(lep['Z']))%10)[:,0:3]"
```

Comment:
Maybe a [qualitative colormap](https://matplotlib.org/examples/color/colormaps_reference.html) would be most appropriate, like tab10 or even tab20.

Reply:
sure